api-controllers/ServiceBrokerApiController.js

```diff
@@ -434,7 +438,10 @@ class ServiceBrokerApiController extends FabrikBaseController {
         timeout_in_sec: CONST.OSB_OPERATION.OSB_SYNC_OPERATION_TIMEOUT_IN_SEC
       }))
       .then(done.bind(this))
-      .catch(NotFound, gone);
+      .catch(NotFound, gone)
```

Comment:
Can we test this scenario and see if/what is the error message ? If message is not set, we should set a message here.

Reply:
Following is test output:
```
$ cf csk bp1 mykey1
Creating service key mykey1 for service instance bp1 as admin_service-fabrik...
The service broker rejected the request to https://service-fabrik-internal.cf.gcp-dev18.gcp.sapcloud.io/cf/v2/service_instances/ac310148-4f8a-41fa-918c-e94f7f80d5b5/service_bindings/4dbbcf43-5dcb-434f-85ed-70746a631674. Status Code: 429 Too Many Requests, Body: {}
FAILED
```
Body is empty here, should we send an error description in body? @subhankarc  @amitmalav

Reply:
IMO it's okay to keep the body empty. `Status Code: 429 Too Many Requests` is anyway shown in the message.

Reply:
yup. Agree.